README.md

```diff
@@ -168,6 +169,20 @@ For example, the following would use `index.ts` as the secondary entry point:
 }
 ```
 
+#### How to embed assets in CSS?
+
+You can embed assets such as font and images inside the outputted css. More information [in the CSS tricks website](https://css-tricks.com/data-uris)
+
+```json
+{
+  "ngPackage": {
+    "lib": {
+      "embedAssets": true
```

Comment:
Added new "config"

Reply:
Can you please explain what you mean please?